server/src/main/java/org/opensearch/gateway/BaseGatewayShardAllocator.java

```diff
@@ -92,6 +92,8 @@ public void allocateUnassigned(
         }
     }
 
+    public void allocateUnassignedBatch(String batchId, RoutingAllocation allocation) {}
```

Comment:
Will remove above Diff, since the above diff is covered in this PR: https://github.com/opensearch-project/OpenSearch/pull/8776/files

The above PR will be merged first